libraries/loader/src/index.js

```diff
@@ -57,15 +57,16 @@ function renderGraph(componentDirectory, sortedComponents) {
  * Usage is detailed in README.
  */
 function loader(componentDirectory, virtualComponents = {}) {
-  assume(componentDirectory).is.an('object');
+  assert(typeof componentDirectory === 'object');
   if (virtualComponents instanceof Array) {
     virtualComponents = virtualComponents.reduce((acc, k) => {
       acc[k] = null;
       return acc;
     }, {});
   }
   const virtualKeys = Object.keys(virtualComponents);
-  assume(Object.keys(componentDirectory).filter(x => virtualKeys.includes(x))).has.length(0);
+  assert(Object.keys(componentDirectory).every(k => !virtualKeys.includes(k)),
```

Comment:
I am confused as to why this is causing tests to fail. To my brain this looks equivalent.

Reply:
It is -- it's just the test was looking for a different error message.  Sorry about that!